disk.lua

```diff
@@ -127,6 +139,8 @@ function disk:update_xml(id, update_function)
                 project_file:close()
             end)
         if not success then
+            if project_file then project_file:close() end
+            ngx.log(error)
             yield_error(err.unparseable_xml .. tostring(message))
         end
```

Comment:
@bromagosa Some thoughts about things that we could do, but are hard to say without a root cause:

* we could read the file back right here, which would be a good check against the 0 byte case. 
* we could still read the file back and ensure that the total bytes are within +-(length(name + description)) of the length we read in. 

`ngx.log(error)` is also wrong. 🤦 It should be `message`. And we probably want to add a call to rollbar if this happens and actually trigger a notification.